index.html

```diff
@@ -177,7 +177,7 @@ <h2>About Me</h2>
                             <img src="images/Icons/icon-languages.svg" alt="icon-languages">
                         </div>
                         <div class="about-datails">
-                            <h2>Languages</h2>
+                            <h2>languages</h2>
```

Comment:
Why don't you start with a capital letter here in respect to your Figma design.😒😒😒😒

Reply:
Dear @Simpleshaikh1 ,

Thanks for your guidance I made all titles in capital letter.